src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -366,6 +366,21 @@ private void CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, Method
 #endif
             }
 
+            if (codeSize != _code.Length)
+            {
+                // If the generated code is smaller than allocatedHotCodeSize, then trim the codeBlock.
+                //
+                // Currently, hot/cold splitting is not done and hence `codeSize` just includes the size of
+                // hotCode. Once hot/cold splitting is done, need to trim respective `_code` or `_coldCode`
+                // accordingly.
+                if (_compilation.Logger.IsVerbose)
+                {
+                    Log.WriteLine($"Trimming codeSize from {_code.Length} to {codeSize}, Savings: {_code.Length - codeSize}, Method: {_methodCodeNode.Method.Name}");
```

Comment:
We should not be logging this to the verbose log.

Reply:
Sounds good. will remove it.